src/Native/Unix/configure.cmake

```diff
@@ -30,6 +30,10 @@ endif ()
 # We compile with -Werror, so we need to make sure these code fragments compile without warnings.
 set(CMAKE_REQUIRED_FLAGS -Werror)
 
+# Older CMake versions (3.8) do not assign the result of their tests, causing unused-value errors
+# which are not distinguished from the test failing.
+set(CMAKE_REQUIRED_FLAGS -Wno-error=unused-value)
```

Comment:
This just overwrites the previous set CMAKE_REQUIRED_FLAGS

Reply:
I guess that caused the alpine build error. The strerror_r is not declared, but C just issues warning, it is not a hard error. Now you've removed the -Werror, so the compilation of the check succeeded.

Reply:
And that's why it broke GLIBC `strerror_r` test that also depends on a subtle compiler warning.

You may change the `strerror_r` error test to: `char c = *strerror_r(0, NULL, 0);` so that it does not depend on the subtle compile warning settings.

Reply:
I see, I"ll fix.
```
test.c:5:19: warning: initialization makes pointer from integer without a cast [-Wint-conversion]
         char* c = strerror_r(0, buffer, 0);
                   ^
```